mobile-app/src/screens/Team/Team.js

```diff
@@ -47,13 +47,18 @@ export default class Team extends Component {
     id === myTeam ? onUpdateProfileTeam({ team: '' }) : onUpdateProfileTeam({ team: id })
   };
 
-  renderInfo = (icon, title, text, arrow, onPress) => (
+  renderInfo = (icon, title, text, arrow, onPress, counter) => (
     <View style={{ marginTop: 15 }}>
       <Text style={styles.infoTitle}>{title.toUpperCase()}</Text>
       <TouchableOpacity style={styles.infoTextWrapper} onPress={onPress}>
         {icon && <Image source={icon} style={styles.image} resizeMode="contain"/>}
         <Text style={styles.text}>{text}</Text>
         {arrow && <Image source={arrow} style={styles.arrow} resizeMode="contain"/>}
+        {!!(counter || counter === 0) &&
```

Comment:
Can counter be non-numeric or why is this check needed?

Reply:
Zero value will be treated as falsy and React will try to render it outside of Text tag, which will cause an error.

Reply:
Sure, but the check allows any truthy value or zero, only excluding null and undefined. If those values are not possible, the whole check is redundant.